internal/driver/glfw/driver_xdg.go

```diff
@@ -0,0 +1,44 @@
+//go:build linux || freebsd || openbsd || netbsd
+
+package glfw
+
+import "C"
+import (
+	"github.com/godbus/dbus/v5"
+
+	"fyne.io/fyne/v2"
+)
+
+var inhibitCookie uint32
+
+func setDisableScreenBlank(disable bool) {
+	conn, err := dbus.SessionBus() // shared connection, don't close
+	if err != nil {
+		fyne.LogError("Unable to connect to session D-Bus", err)
+		return
+	}
+
+	if !disable {
+		if inhibitCookie != 0 {
+			obj := conn.Object("org.freedesktop.ScreenSaver", "/org/freedesktop/ScreenSaver")
+			call := obj.Call("org.freedesktop.ScreenSaver.UnInhibit", 0, inhibitCookie)
+			if call.Err != nil {
+				fyne.LogError("Failed to send message to bus", call.Err)
+			}
+			inhibitCookie = 0
+		}
+		return
+	}
+
+	if inhibitCookie != 0 {
+		return
+	}
+	obj := conn.Object("org.freedesktop.ScreenSaver", "/org/freedesktop/ScreenSaver")
+	call := obj.Call("org.freedesktop.ScreenSaver.Inhibit", 0, fyne.CurrentApp().Metadata().Name,
+		"App disabled screensaver")
```

Comment:
What kind of standard do we have in regards to strings being localized now that we support that?

Reply:
Good point. It could be localised, but this felt like it was developer info not user info - am I wrong about that too?

Reply:
Yeah, I didn't seem to get a notification from it when enabling inhibit so maybe it isn't even always presented to the user? I think we can avoid localising it for now :)